ibis/backends/flink/registry.py

```diff
@@ -17,6 +23,12 @@
 operation_registry = base_operation_registry.copy()
 
 
+def type_to_sql_string(tval):
+    if tval.is_array():
+        return f"array<{helpers.type_to_sql_string(tval.value_type)}>"
+    return helpers.type_to_sql_string(tval)
```

Comment:
Eventually we should use the SchemaMapper and TypeMapper classes for this kind of thing, but not a blocker for this PR.